superset/datasets/dao.py

```diff
@@ -79,7 +79,7 @@ def validate_table_exists(database: Database, table_name: str, schema: str) -> b
             database.get_table(table_name, schema=schema)
             return True
         except SQLAlchemyError as ex:  # pragma: no cover
-            logger.error("Got an error %s validating table: %s", str(ex), table_name)
+            logger.info("Got an error %s validating table: %s", str(ex), table_name)
```

Comment:
since validation is expected to fail sometimes, it seems appropriate to log this a info instead of error.

Reply:
warn?